haskell-org-committee.markdown

```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Discourse][discourse]
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
+
+As a whole it seeks to service the open source Haskell community.
+
+Some particular aspects of the work of Haskell.org include producing and maintaining material regarding the Haskell language on haskell.org and other associated websites; hosting and maintaining shared assets and infrastructure for the benefit of the Haskell community; coordinating educational activities including mentorship programs; and potentially organizing event regarding the Haskell language.
+
+Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][gsoc].
+
+### Contact
+
+  * [committee@haskell.org][email]: directly email the committee
+  * [haskell-community mailing list][list]: public discussions about the committee and Haskell community infrastructure
+  * [GitHub][github]: issues, suggestions and pull requests for the website
+  * TODO: add one for proposal process?
```

Comment:
Adding a link to the proposals repo is a great idea. I guess we should also write up a description of the process in that repo's README or something...

Reply:
@TikhonJelvis, sorry for being a derp, but where is that proposals repo? I've been unable to locate it.

Reply:
Oh, I found it referenced in another PR - https://github.com/haskell-org/committee. I'll add that now.

Reply:
oh, this is already added, resolving.